# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perl/use_db3.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Force perl to use bdb33, for on-disk data binary compatibility reasons.

  - Rene Rebe <rene@exactcode.de>

--- perl-5.8.7/ext/DB_File/Makefile.PL.orig	2002-03-25 00:13:47.000000000 +0100
+++ perl-5.8.7/ext/DB_File/Makefile.PL	2005-08-07 20:58:46.000000000 +0200
@@ -8,20 +8,19 @@
 # OS2 is a special case, so check for it now.
 my $OS2 = "-DOS2" if $Config{'osname'} eq 'os2' ;
 
-my $LIB = "-ldb" ;
+my $LIB = "-ldb-3" ;
 # so is win32
 $LIB = "-llibdb" if $^O eq 'MSWin32' ;
 
 WriteMakefile(
 	NAME 		=> 'DB_File',
-	LIBS 		=> ["-L/usr/local/lib $LIB"],
+	LIBS 		=> ["-L/usr/lib $LIB"],
         MAN3PODS        => {},         # Pods will be built by installman.
-	#INC		=> '-I/usr/local/include',
 	VERSION_FROM	=> 'DB_File.pm',
 	OBJECT		=> 'version$(OBJ_EXT) DB_File$(OBJ_EXT)',
 	XSPROTOARG	=> '-noprototypes',
 	DEFINE		=> $OS2 || "",
-	INC		=> ($^O eq "MacOS" ? "-i ::::db:include" : ""),
+	INC		=> ($^O eq "MacOS" ? "-i ::::db:include" : "-I/usr/include/db3"),
 	'depend'	=> {'version$(OBJ_EXT)' => 'version.c'},
 	'clean'		=> {FILES => 'constants.h constants.xs'},
 	);
